apps/guardian-ui/src/components/SetConfiguration.tsx

```diff
@@ -239,7 +239,11 @@ export const SetConfiguration: React.FC<Props> = ({ next }: Props) => {
             onChange={(ev) => setPassword(ev.currentTarget.value)}
             isDisabled={!!statePassword}
           />
-          <FormHelperText>{t('set-config.admin-password-help')}</FormHelperText>
+          <FormHelperText>
+            <Text color={theme.colors.yellow[400]}>
```

Comment:
Contrast ratio on this is pretty bad, might be worth bumping to 500 for text on white.

![CleanShot 2024-01-09 at 16 17 33@2x](https://github.com/fedimint/ui/assets/649992/c85d058f-6e36-4656-8839-a3fe13c51587)

Reply:
switched to 500 in next commit